### PR TITLE
Improvements to batch releasing

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: PROCESSES.md [hunk 1]
```diff
@@ -207,13 +207,15 @@ you need to.
...
-environment variable to authenticate with the API, so make this is
-set beforehand. The access token should include the `repo` scope.
+create a pull request with the `autorelease: pending` and
+`automerge: exact` labels. If the `RELEASE_PR_ASSIGNEE` environment
+variable is set, the PR is created with that assignee. Note that
+this command checks that there are no project references from APIs
+being released now to APIs that *aren't* being released. Without
+this check, it's possible for a released version to depend on
+unreleased changes. This uses the `GITHUB_ACCESS_TOKEN` environment
+variable to authenticate with the API, so make this is set
```
Comment (amanda-tarafa) [inline PROCESSES.md:217]:
```suggestion
variable to authenticate with the API, so make sure this is set
```

File: tools/Google.Cloud.Tools.ReleaseManager/BatchRelease/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+The following JSON shows the general options, with their default values.
+
+```jsonc
+{
+  // When true, each release (and history) is confirmed manually at the console.
+  "ConfirmRelease": true,
+
+  // Whether or not to rewrite the root README.md file for each release.
+  // This is normally a good idea, but if you're releasing a APIs which are adjacent
```
Comment (amanda-tarafa) [inline tools/Google.Cloud.Tools.ReleaseManager/BatchRelease/README.md:37]:
```suggestion
  // This is normally a good idea, but if you're releasing APIs which are adjacent
```

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Comment by jskeet** [comment]
Changes made - will merge on green.

**Pull request merged**